morph.js

```diff
@@ -2257,12 +2256,14 @@ export class Morph {
     this.position = dragStartMorphPosition.addPt(absDragDelta);
   }
 
-  onDragEnd(evt) {
+  async onDragEnd(evt) {
+    let { removeSnapToGuidesOf } = await System.import("lively.halos/drag-guides.js");
```

Comment:
Looks OK to me but actually, we should be fine with simply adding this as a normal import.  In case we don't want to include it we can simply map that to an `@empty` module via `System.config({map...})` or in the compiler when building a release.  The code then needs to deal with the absence of halos and such of course.

Reply:
Ok, please take a look at the lastest commit 565760a where I replaced all of the System.imports with our default imports. Do we already have a compiler mechanism for releases? Maybe I should push my boilerplate code for static compilation from a few weeks ago. But that stuff assumes still file based modules, not the lively.storage based system we have now. That requires a little fixing, I think.

Reply:
Already, let's try that out.

Can you post your extraction code somewhere? Gist or a lively-next world e.g?

Reply:
https://gist.github.com/merryman/40dccccc406938e4ad7efad24e54c038